lib/bootloader_setup.pm

```diff
@@ -647,14 +648,17 @@ sub autoyast_boot_params {
         $autoyast_args .= "$proto://10.0.2.1/";
         $autoyast_args .= 'data/' if $ay_var !~ /^aytests\//;
         $autoyast_args .= $ay_var;
-    } elsif ($ay_var =~ /^ASSET_\d+$/) {
+    }
+    elsif ($ay_var =~ /^ASSET_\d+$/) {
```

Comment:
Why need change  `} elsif ` to `}\n elseif {` ?

Reply:
Does the pertuidy do it?

Reply:
yup, Perltidy changed the style... let me mark it resolved for the time being